addons/sale_loyalty/models/sale_order.py

```diff
@@ -220,7 +220,7 @@ def _get_reward_values_product(self, reward, coupon, product=None, **kwargs):
             'points_cost': cost,
             'reward_identifier_code': _generate_random_reward_code(),
             'sequence': max(self.order_line.filtered(lambda x: not x.is_reward_line).mapped('sequence'), default=10) + 1,
-            'tax_id': [(Command.CLEAR, 0, 0)] + [(Command.LINK, tax.id, False) for tax in taxes]
+            'tax_ids': [(Command.CLEAR, 0, 0)] + [(Command.LINK, tax.id, False) for tax in taxes]
```

Comment:
```suggestion
            'tax_ids': [Command.clear()] + [Command.link(tax.id) for tax in taxes]
```
to reduce a bit the line length since you're already modifying it :eyes: